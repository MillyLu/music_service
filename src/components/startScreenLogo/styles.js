import styled from "styled-components";


const NavLogo = styled.div`
width: 113.33px;
  height: 43px;
  padding: 13px 0 13px 0;
  background-color: transparent;
  margin-bottom: 20px;
`;

export const LogoImage = styled.img`
width: 113.33px;
height: 17px;
color: #181818;
`;


export const StartscreenDivLogo = styled(NavLogo)`
width: 140px;
height: 21px;
padding: 0;
margin-button: 42px;
background-color: transparent;
`

export const StartscreenLogo = styled(LogoImage)`
width: 140px;
height: 21px;
color: black;
`;