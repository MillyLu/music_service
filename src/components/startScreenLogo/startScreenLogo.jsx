import logo from './logo-log.png';
import * as Styled from './styles';

export function StartscreenLogo () {
    return(
        <Styled.StartscreenDivLogo>
            <Styled.StartscreenLogo src={logo}/>
        </Styled.StartscreenDivLogo>
        
    )

}