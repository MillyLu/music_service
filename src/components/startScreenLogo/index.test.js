/* eslint-disable no-undef */
import * as ShallowRenderer from 'react-shallow-renderer';
// import { ShallowRenderer, createRenderer } from 'react-shallow-renderer';
import { StartscreenLogo } from './startScreenLogo';



describe('<StartscreenLogo/>', () => {
        it('should render successfully and match snapshot', () => {
            const shallowRenderer = ShallowRenderer.createRenderer();
            const renderer = shallowRenderer;
            renderer.render(<StartscreenLogo />);
            const result = renderer.getRenderOutput();
            expect(result).toMatchSnapshot();
          })
      });