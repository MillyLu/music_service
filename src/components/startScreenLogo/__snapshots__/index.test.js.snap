// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StartscreenLogo/> should render successfully and match snapshot 1`] = `
<Styled(styled.div)>
  <Styled(styled.img)
    src="logo-log.png"
  />
</Styled(styled.div)>
`;
